R/load.r

```diff
@@ -148,12 +150,13 @@ load_all <- function(path = ".", reset = TRUE, recompile = FALSE,
     unload_dll(package)
   }
 
-  if (recompile) {
-    pkgbuild::clean_dll(path)
+  # Compile dll if requested
+  if (!missing(recompile) && missing(compile)) {
```

Comment:
Do we still need to clean the existing objects with `pkgbuild::clean_dll()` if recompile = TRUE?

Actually maybe we should remove `recompile` and add a `clean` argument, then the logic can be

```r
if (isTRUE(clean)) {
  pkgbuild::clean_dll(path)
}

if (isTRUE(compile)) {
  pkgbuild::compile_dll(path)
}
```

Which seems easier to understand to me.

Reply:
Oh, that's the problem with changing the default argument — `NA` means build if needed, and we now don't have a way to force a rebuild.  

I personally prefer that to `clean` + `compile`